src/bio/bio_device.c

```diff
@@ -673,3 +675,139 @@ bio_dev_list(struct bio_xs_context *xs_ctxt, d_list_t *dev_list, int *dev_cnt)
 
 	return rc;
 }
+
+int
+bio_set_led_state(struct bio_xs_context *xs_ctxt, uuid_t dev_uuid,
+		  const char *led_state, bool reset)
+{
+	struct spdk_pci_addr	pci_addr;
+	struct spdk_pci_device *pci_device;
+	struct bio_bdev	       *bio_dev;
+	struct bio_dev_info	b_info = { 0 };
+	enum spdk_vmd_led_state current_led_state;
+	int			new_led_state;
+	int			rc = 0;
+	bool			found = false;
+
+	D_ASSERT(is_init_xstream(xs_ctxt));
+
+	bio_dev = lookup_dev_by_id(dev_uuid);
+	if (bio_dev == NULL) {
+		D_ERROR("Failed to find dev "DF_UUID"\n",
+			DP_UUID(dev_uuid));
+		return -DER_NONEXIST;
+	}
+
+	/* LED wil be reset to the orignal saved state */
```

Comment:
There is a typo in this comment that causes checkpatch to fail:
https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-3950/9/execution/node/60/log/?consoleFull
21:40:25  src/bio/bio_device.c:701: wil ==> will, well

Reply:
@mjmac just pushed typo fix